cost/aws/reserved_instances/expiration/expired_ris.pt

```diff
@@ -8,34 +8,29 @@ category "Cost"
 tenancy "single"
 default_frequency "daily"
 info(
-  version: "2.1",
-  provider: "Flexera Optima",
-  service: "",
-  policy_set: ""
+  version: "2.2",
+  provider: "AWS",
+  service: "Compute",
+  policy_set: "Reserved Instances"
 )
 
 ###############################################################################
-# Permissions
+# Parameters
 ###############################################################################
 
-permission "optima" do
-  label "Access Optima Resources"
-  resources "rs_optima.aws_reserved_instances"
-  actions "rs_optima.index"
+parameter "param_email" do
+  type "list"
+  category "Policy Settings"
+  label "Email Addresses"
+  description "Email addresses of the recipients you wish to notify when new incidents are created."
+  default []
 end
 
-###############################################################################
-# User inputs
-###############################################################################
-
 parameter "param_heads_up_days" do
   type "number"
-  label "Number of days to prior to expiration date to trigger incident"
-end
-
-parameter "param_email" do
-  type "list"
-  label "Email addresses of the recipients you wish to notify"
+  category "Policy Settings"
+  label "Days Prior To Expiration"
+  description "Number of days to prior to expiration date to trigger incident"
```

Comment:
@XOmniverse - should this have a default number of days?

I noticed the same for AWS Savings Plan Expiration, which previously had a default number of days equal to 7, but has been removed with this change.

Reply:
Will address this in future changes to avoid too many changes as part of this PR that modify policy functionality.